node/core/backing/src/lib.rs

```diff
@@ -680,31 +680,41 @@ impl CandidateBackingJob {
 
 		let summary = self.table.import_statement(&self.table_context, stmt);
 
-		if let Some(ref summary) = summary {
-			if let Some(attested) = self.table.attested_candidate(
-				&summary.candidate,
-				&self.table_context,
-			) {
-				// `HashSet::insert` returns true if the thing wasn't in there already.
-				// one of the few places the Rust-std folks did a bad job with API
-				if self.backed.insert(summary.candidate) {
-					self.remove_unbacked_span(&summary.candidate);
-
-					if let Some(backed) =
-						table_attested_to_backed(attested, &self.table_context)
-					{
-						let message = ProvisionerMessage::ProvisionableData(
-							self.parent,
-							ProvisionableData::BackedCandidate(backed.receipt()),
-						);
-						self.send_to_provisioner(message).await?;
-					}
+		let unbacked_span = if let Some(attested) = summary.as_ref()
+			.and_then(|s| self.table.attested_candidate(&s.candidate, &self.table_context))
+		{
+			let candidate_hash = attested.candidate.hash();
+			// `HashSet::insert` returns true if the thing wasn't in there already.
+			if self.backed.insert(candidate_hash) {
+				let span = self.remove_unbacked_span(&candidate_hash);
+
+				if let Some(backed) =
+					table_attested_to_backed(attested, &self.table_context)
+				{
+					let message = ProvisionerMessage::ProvisionableData(
+						self.parent,
+						ProvisionableData::BackedCandidate(backed.receipt()),
+					);
+					self.send_to_provisioner(message).await?;
+
+					span.as_ref().map(|s| s.child("backed"));
```

Comment:
are we creating and dropping a child span here? this warrants some comments

Reply:
Why does this warrants a comment? 

It should just add a span says "backed" to see in the UI when a unbacked candidate was backed.

Reply:
it's not obvious that we encode a state transition with an ephemeral child span that is dropped immediately, I'm not sure this is the intended use of spans

Reply:
The whole "unbacked" story is probably not the intended use of spans. :shrug: